.github/workflows/ci.yml

```diff
@@ -23,7 +23,7 @@ jobs:
 
     steps:
       - name: Checkout code
-        uses: actions/checkout@v1
+        uses: actions/checkout@v3
 
       - name: Setup python
         uses: actions/setup-python@v2
```

Comment:
I notice we're getting some warnings related to `actions/setup-python@v2`. Those can be addressed separately if that's easier.

> Node.js 12 actions are deprecated. Please update the following actions to use Node.js 16: actions/setup-python@v2. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/.

Reply:
Yep, I wanted to take those separately, because latest version was v4 and I still need to go over the changelog.